scripts/setup_irods.py

```diff
@@ -150,13 +148,10 @@ def setup_server(irods_config, json_configuration_file=None, test_mode=False):
     test_put(irods_config)
 
     l.info(irods.lib.get_header('Log Configuration Notes'))
-    l.info(('The iRODS log file is managed by rsyslog and logrotate.\n'
-            'The locations of the log file and configuration files are listed below.\n\n'
-            '  Log File Path               : ' + irods.paths.server_log_path() + '\n'
-            '  Rsyslog Configuration Path  : ' + rsyslog_config_path() + '\n'
-            '  Logrotate Configuration Path: ' + logrotate_config_path() + '\n\n'
-            'iRODS will never touch these configuration files again.\n'
-            'If you need to make adjustments, you must do so manually.'))
+    l.info(('iRODS uses syslog for logging. If your OS has a running syslog service, messages\n'
+            'will appear in the system log file (normally located at /var/log/syslog).\n\n'
+            'See the following for more information:\n\n'
+            f'  https://docs.irods.org/{version_json["irods_version"]}/system_overview/troubleshooting/'))
```

Comment:
Any thoughts on the wording?

The location of the docs is temporary (because I haven't wrote the docs yet).

Reply:
Wording is ... good enough for now I think (we haven't heard anyone who is unhappy seeing it yet, or having trouble).

I still think this is probably the right approach - but we definitely want to make it as painless/obvious as possible for people to get their iRODS logs in one place, rather than the system log file.   We will certainly get email about this unless the process is smooth / obvious / good.

Perhaps something like...
"See the following for more information about configuring an iRODS-specific logfile:"

Reply:
Agreed.

Reply:
Done.